src/util/redirects-raw.txt

```diff
@@ -10,6 +10,7 @@
 /docs/workflows/storybook-composition/           /docs/sharing/storybook-composition/                                           301
 /docs/workflows/package-composition/             /docs/sharing/package-composition/                                             301
 /docs/workflows/faq/                             /docs/faq/                                                                     301
+/docs/addons/addon-catalog                       /docs/integrations/integration-catalog                                         301
```

Comment:
@Integrayshaun — Because of the framework that's part of `/docs/...` URLs, we have to handle those redirects a bit differently. The entries in this file run through this function to generate all of the variations necessary: https://github.com/storybookjs/frontpage/blob/main/gatsby-node.js#L250-L298